tests/framework/microvm.py

```diff
@@ -552,6 +589,43 @@ def start(self):
         response = self.actions.put(action_type='InstanceStart')
         assert self._api_session.is_status_no_content(response.status_code)
 
+    def pause_to_snapshot(self,
+                          mem_file_path=None,
```

Comment:
Should we validate that the paths are not `None`?

Reply:
I would like it to be None in a negative test that we need to implement.

Reply:
Maybe we should log an issue for negative testing the Snapshot API so we don't forget.

Reply:
We are tracking this here: https://github.com/firecracker-microvm/firecracker/issues/2005